### PR TITLE
Add crypto command

### PR DESCRIPTION
**Pull request by Sata51**
⚠️  Untested :D

Syntax:
- !crypt [crypto currency] [real currency]

Usage: 
- !crypto BTC
- !crypto BTC USD
- !crypto BTC EUR

### CODE REVIEW

File: src/bot/commands/crypto.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { command } from '../lib/utils/command.js'
+import price from 'crypto-price'
+
+async function resolver({ command: { args } }) {
+  let [cryptoCurrency, currency] = args
+
+  if (currency === null) {
+    currency = 'USD'
```
Comment (WinXaito) [inline src/bot/commands/crypto.js:8]: Please put `CHF` by default :)

Comment (Sata51) [reply src/bot/commands/crypto.js:8]: NO

File: src/bot/commands/crypto.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  }
+  if (cryptoCurrency === null) {
+    cryptoCurrency = 'BTC'
```
Comment (mindsers) [inline src/bot/commands/crypto.js:11]: Why not use a default value on the destructuration line 5?

**Review by mindsers — APPROVED** [review]
👍🏽

**Pull request merged**